plugins/controlboard/src/ControlBoardData.cpp

```diff
@@ -0,0 +1,114 @@
+
+#include "ControlBoardData.hh"
```

Comment:
Just to be consistent with the codebase

```suggestion
#include <ControlBoardData.hh>
```